src/popover/popover.ts

```diff
@@ -18,10 +19,16 @@ import {
   NgZone,
   SimpleChanges
 } from '@angular/core';
+import {DOCUMENT} from '@angular/common';
+import {fromEvent, race} from 'rxjs';
+import {filter, takeUntil} from 'rxjs/operators';
 
-import {listenToTriggers} from '../util/triggers';
+import {listenToTriggers, parseTriggers} from '../util/triggers';
```

Comment:
`parseTriggers` and `isDefined` are unused

Reply:
Forgotten leftover, I'll remove